configs/_base_/trainers/base.py

```diff
@@ -10,14 +10,15 @@
     devices=-1,
     gradient_clip_val=0.5,
     log_every_n_steps=10,
-    val_check_interval=5000,
+    val_check_interval=1000,

```

Comment:
I don't think changing val_check_interval to 1000 is necessary. It will produce lots of testing wavs and fill out the disk.